package custom;

message hello_message {
    optional string text = 1;
}
